client/src/partials/templates/bhDateEditor.tmpl.html

```diff
@@ -0,0 +1,37 @@
+<!-- Date editor component view implementation-->
+<div 
+	class="form-group"	
+	ng-class="{'has-error' : $ctrl.validationTrigger && $ctrl.form.dateEditorValue.$invalid}" >
+
+	<!-- Input component implementation-->
+	<div class="input-group">
+	    <input
+	    	data-date-editor-input
+	    	name="dateEditorValue"
+	    	type="text"
+	    	class="form-control"	    	
+	    	uib-datepicker-popup="{{$ctrl.dateFormat}}"
+	    	ng-model="$ctrl.dateValue"
+	    	is-open="$ctrl.editMode"
+	    	min-date="$ctrl.minDate"
+	    	max-date="$ctrl.maxDate"
+	    	close-text="Close"
+	    	ng-disabled="!$ctrl.editMode"
+	    	show-button-bar="true"
+	    	required
+	    	/>
+	    <span class="input-group-btn">
+	    	<button id="set-date-btn" class="btn btn-default" type="button" ng-click="$ctrl.changeMode()" ng-disabled="$ctrl.editMode">
+	    		{{"DATEFINDER.SET" | translate}}
+	    	</button>
+	    </span>
+    </div> 
+
+    <!-- Error message implementation-->
+    <p class="help-block" ng-messages="$ctrl.form.dateEditorValue.$error" ng-show="$ctrl.validationTrigger">
```

Comment:
Because : 
1. This code looks clear
2. ``` <div ng-messages-include="partials/templates/messages.tmpl.html"></div>```  is only in English since our system is designed to support many languages
3. There is only two translations statements in the component.

Adding peaces of file like this without creating an issue and waiting for an agreement (***I don't know if this template was adopted or agreed***), restrict the developer freedom and lead him to learn additional things which can be probably not useful or productive (in case the team notice that the solution was not profitable).
The date editor component is handling error him self to be more accurate, no need of generic template.

**We are risking to fall into the trap of modularity**